apps/vc-api/docs/tutorials/consent-and-issuance-credential-tutorial.md

```diff
@@ -0,0 +1,1701 @@
+the site. It should looke similar to `https://webhook.site/efb19fb8-2579-4e1b-8614-d5a03edaaa7a`
+Copy this URL, including the domain, into the exchange definition below.
+
+```json
+{
+   "exchangeId":"<SOME UNIQUE ID>",
+   "query":[
+         {
+            "type":"PresentationDefinition",
+            "credentialQuery":[
+               {
+                  "presentationDefinition":{
+                     "id":"286bc1e0-f1bd-488a-a873-8d71be3c690e",
+                     "input_descriptors":[
+                        {
+                           "id":"PermanentResidentCard",
+                           "name":"PermanentResidentCard",
+                           "purpose":"PermanentResidentCard",
+                           "constraints":{
+                              "fields":[
+                                 {
+                                    "path":[
+                                       "$.type"
+                                    ],
+                                    "filter":{
+                                       "type":"array",
+                                       "contains":{
+                                          "type":"string",
+                                          "const":"PermanentResidentCard"
+                                       }
+                                    }
+                                 }
+                              ]
+                           }
+                        }
+                     ]
+                  }
+               }
+            ]
+         },
+         {
+            "type":"PresentationDefinition",
+            "credentialQuery":[
+               {
+                  "presentationDefinition":{
+                     "id":"286bc1e0-f1bd-488a-a873-8d71be3c690e",
+                     "input_descriptors":[
+                        {
+                           "id":"consent_credential",
+                           "name":"Consent Credential",
+                           "purpose":"One consent credential is must for this presentation",
+                           "constraints":{
+                              "fields":[
+                                 {
+                                    "path":[
+                                       "$.type"
+                                    ],
+                                    "filter":{
+                                       "type":"array",
+                                       "contains":{
+                                          "type":"string",
+                                          "const":"ConsentCredential"
+                                       }
+                                    }
+                                 }
+                              ]
+                           }
+                        }
+                     ]
+                  }
+               }
+            ]
+         }
+      ],
```

Comment:
I think there should only be one `presentationDefinition` element in the `credentialQuery` (so that only one VP needs to be created and only one user signature/confirmation is required)

Reply:
While keeping both the input descriptors in one `presentationDefinition`, the VP verification failed. After multiple arrangements the above worked (You can see the output of the callback notification, it is what we expect). 
I agree with you and logically it should be just one presentationDefinition. I will try to find a way to only use one presentationDefinition.